code/game/objects/structures/signs.dm

```diff
@@ -8,6 +8,22 @@
 	armor = list(MELEE = 50, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, RAD = 0, FIRE = 50, ACID = 50)
 	flags_2 = RAD_PROTECT_CONTENTS_2 | RAD_NO_CONTAMINATE_2
 	blocks_emissive = EMISSIVE_BLOCK_GENERIC
+	var/does_emissive = FALSE
+
+/obj/structure/sign/Initialize(mapload)
+	. = ..()
+	if(does_emissive)
+		update_icon()
+		set_light(1, 0.1)
+
+/obj/structure/sign/update_overlays()
+	. = ..()
+
+	if(!does_emissive)
+		return
+	underlays.Cut()
```

Comment:
```suggestion
	underlays.Cut()
	if(!does_emissive)
		return
```
Prob want to switch them up in case later `does_emissive` changes. Prob no function that is in the code yet though